external/webserver/http_client.c

```diff
@@ -582,6 +582,15 @@ int http_recv_and_handle_request(struct http_client_t *client, struct http_keyva
 	conn_type = http_keyvalue_list_find(request_params, "Connection");
 	if (!strncasecmp(conn_type, "Keep-Alive", strlen("Keep-Alive")+1)) {
 		client->keep_alive = 1;
+		struct timeval tv;
+		tv.tv_sec = HTTP_CONF_KEEP_ALIVE_CONN_SOCKET_TIMEOUT_MSEC / 1000;
+		tv.tv_usec = (HTTP_CONF_KEEP_ALIVE_CONN_SOCKET_TIMEOUT_MSEC % 1000) * 1000;
+		HTTP_LOGD("Keep-alive case, change timeout to (%d.%d)sec\n", tv.tv_sec, tv.tv_usec);
```

Comment:
`tv_sec` is unsigned type, so I think it is better to use `%u` instead of `%d`.

Reply:
change done